rpcserver.go

```diff
@@ -5706,11 +5706,12 @@ func (r *rpcServer) DeleteAllPayments(ctx context.Context,
 	req *lnrpc.DeleteAllPaymentsRequest) (
 	*lnrpc.DeleteAllPaymentsResponse, error) {
 
-	rpcsLog.Debugf("[DeleteAllPayments] failed_payments_only=%v",
-		req.FailedPaymentsOnly)
+	rpcsLog.Debugf("[DeleteAllPayments] failed_payments_only=%v, "+
```

Comment:
I think this could be raised to info level since it's not common and only one line per RPC call.

Reply:
Agreed 👍